ops/mainnet/prod/core/config.tf

```diff
@@ -366,6 +366,10 @@ locals {
       "6778479"    = 10,
       "1818848877" = 10
     }
+    proverBatchWaitTime = {
+      "6648936" = 14400,
```

Comment:
We should change this 12 hours for Mainnet, and communicate to mods that, in rare cases it might take 12 hours, for a slow path transfer to complete.

We should have 4 hours for all other chains, by default, since if the batch is fully by then, it will still submit before that time.

Reply:
updated